R/utils.R

```diff
@@ -268,15 +268,26 @@ auto_font <- function(txt, txt_max = 45, size_min = .75, size_max = 1.5,
 
 
 #' The 'Build Community plot' function
+#' @description 
+#' build_comm_plotly() is responsible for building the plotly graphic of community usage metrics
 #' 
-#' @param data a data.frame
-#' 
+#' @param data a data.frame that is built in databaseView.R from get_comm_data()
```

Comment:
You have to remember that this is an exported function, so this parameter description won't make any sense to our users. Try:

```suggestion
#' @param data a data.frame containing monthly download data, built using `generate_comm_data()`. This argument is optional, but if `NULL`, a `pkg_name` must be provided.
```

Reply:
@AARON-CLARK Updated